packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -404,3 +531,60 @@ class GroupPage extends React.Component {
   }
 }
 ```
+
+### Vertical nav using PageHeader component
```

Comment:
Can we add a blurb about suggested usage of PageHeader? Is it going to be deprecated?